package.json

```diff
@@ -77,7 +77,7 @@
   ],
   "dependencies": {
     "global": "^4.3.2",
-    "video.js": "^6 || ^7"
+    "video.js": "^7 || ^8"
```

Comment:
The other changes would only be compatible with 8

Reply:
Good call, I removed the v7 option here